website/source/docs/concepts/ha.html.md

```diff
@@ -76,27 +76,27 @@ always required for all HA setups.
 Some HA data store drivers can autodetect the redirect address, but it is often
 necessary to configure it manually via setting a value in the `storage`
```

Comment:
Might want to change this bit since it's no longer in the storage block

Reply:
I updated the wording, let me know what you think!